.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,62 @@
+name: Report a bug
+description: Tell us about a bug or issue you may have identified in Boosted.
+title: "Provide a general summary of the issue"
+labels: [bug]
+assignees: "-"
+body:
+  - type: checkboxes
+    attributes:
+      label: Prerequisites
+      description: Take a couple minutes to help our maintainers work faster.
+      options:
+        - label: I have [searched](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
+          required: true
+        - label: I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
+          required: true
+        - label: I have read the [contributing guidelines](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/.github/CONTRIBUTING.md)
+          required: true
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: Describe the issue
+      description: Provide a summary of the issue and what you expected to happen, including specific steps to reproduce.
+    validations:
+      required: true
+  - type: textarea
+    id: reduced-test-case
+    attributes:
+      label: Reduced test cases
+      description: Include links [reduced test case](https://css-tricks.com/reduced-test-cases/) links or suggested fixes using [CodePen](https://codepen.io/).
```

Comment:
We don't have a [CodePen Teams](https://codepen.io/features/teams) account so we can't do the same thing than Bootstrap: `using CodePen ([v4 template](https://codepen.io/team/bootstrap/pen/yLabNQL) or [v5 template](https://codepen.io/team/bootstrap/pen/qBamdLj)).` which is nice. When this PR will merged, I will track this as an issue and see if we could have a [CodePen Teams](https://codepen.io/features/teams).